modules/setting-up-a-prometheus-server-scraping-from-central-service.adoc

```diff
@@ -0,0 +1,66 @@
+// module included in the following assemblies:
+//
+// * configuration/monitor-acs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="setting-up-a-prometheus-server-scraping-from-central-service_{context}"]
+= Setting up a Prometheus server scraping from Central service
+
+Before connecting Perses or other monitoring tools, you must configure Prometheus to scrape metrics from the Central service. This involves creating a service account, performing role mapping, and assigning the necessary permissions. Completing these steps ensures Prometheus can securely access Central API data for monitoring and visualization purposes.
```

Comment:
```suggestion
Before connecting Perses or other monitoring tools, you must configure Prometheus to scrape metrics from the Central service.

This could be achieved, for example, by configuring the Machine access configuration integration for Kubernetes service account. This involves creating a service account, performing role mapping, and assigning the necessary permissions. Completing these steps ensures Prometheus can securely access Central API data for monitoring and visualization purposes.
```